src/community/mbstyle/src/test/java/org/geoserver/community/mbstyle/web/MBStyleEditPageTest.java

```diff
@@ -0,0 +1,96 @@
+import org.geoserver.community.mbstyle.MBStyleHandler;
+import org.geoserver.web.GeoServerWicketTestSupport;
+import org.junit.Before;
+import org.junit.Test;
+import org.geoserver.wms.web.data.StyleEditPage;
+import org.geotools.styling.ExternalGraphic;
+import org.geotools.styling.PointSymbolizer;
+import org.geotools.styling.PointSymbolizerImpl;
+import org.geotools.styling.SLD;
+import org.geotools.styling.StyledLayerDescriptor;
+import org.geoserver.wms.web.data.OpenLayersPreviewPanel;
+
+
+public class MBStyleEditPageTest extends GeoServerWicketTestSupport {
+    
+	StyleInfo mbstyle;
+	StyleEditPage edit;
+
+	@Before
+	public void setUp() throws Exception {
+	    Catalog catalog = getCatalog();
+	    login();
+	    
+	    mbstyle = new StyleInfoImpl(null);
+	    mbstyle.setName("mbstyle");
+	    mbstyle.setFilename("mbstyle.json");
+	    mbstyle.setFormat(MBStyleHandler.FORMAT);
+	    catalog.add(mbstyle);
+	    mbstyle = catalog.getStyleByName("mbstyle");
+	    catalog.save(mbstyle);
+	    
+	    edit = new StyleEditPage(mbstyle);
+	    tester.startPage(edit);
+    }
+    
+  @Test
+  public void testMbstyleChange() throws Exception {
+
+      String json =
+              "{\n" +
+                      "  \"version\": 8, \n" +
+                      "  \"name\": \"places\",\n" +
+                      "  \"sprite\": \"http://localhost:8080/geoserver/styles/mbsprites\",\n" +
+                      "  \"layers\": [\n" +
+                      "    {\n" +
+                      "      \"id\": \"circle\",\n" +
+                      "      \"source-layer\": \"Buildings\",\n" +
+                      "      \"type\": \"symbol\",\n" +
+                      "      \"layout\": {\n" +
+                      "        \"icon-image\": \"circle\",\n" +
+                      "        \"icon-size\": {\n" +
+                      "          \"property\": \"POP_MAX\",\n" +
+                      "          \"type\": \"exponential\",\n" +
+                      "          \"stops\": [\n" +
+                      "            [0, 0.7],\n" +
+                      "            [40000000, 3.7]\n" +
+                      "          ]\n" +
+                      "        }\n" +
+                      "      }\n" +
+                      "    }\n" +
+                      "  ]\n" +
+                      " }\n";
+      
+      FormTester form = tester.newFormTester("styleForm");
+      form.setValue("context:panel:format", MBStyleHandler.FORMAT);
+      form.setValue("context:panel:name", "mbstyleTest");
+      tester.executeAjaxEvent("apply", "click");
+      tester.executeAjaxEvent("styleForm:context:tabs-container:tabs:2:link", "click");
+      tester.assertComponent("styleForm:context:panel", OpenLayersPreviewPanel.class);
+      OpenLayersPreviewPanel previewPanel = (OpenLayersPreviewPanel) tester.getComponentFromLastRenderedPage("styleForm:context:panel");
+      assertFalse(previewPanel.isPreviewStyleGroup);
```

Comment:
You can just use the value of the checkbox instead; something like: 
```
tester.assertModelValue("styleForm:context:panel:previewStyleGroup", false);
```
(This gets around needing to make `isPreviewStyleGroup a public field)